src/main/webapp/assets/modules/post-contest/postContestController.js

```diff
@@ -40,14 +40,18 @@ techlooper.controller("postContestController", function ($scope, $http, jsonValu
             return moment().add(63, 'day').format('DD/MM/YYYY');
 
           case "start-date-wt-4w":
+            if (!$scope.contest.startDate) return true;
             var lastDate = moment().add(4, 'weeks');
-            return moment($scope.contest.startDate, jsonValue.dateFormat).isBetween(moment(), lastDate, 'day');
+            var startDate = moment($scope.contest.startDate, jsonValue.dateFormat);
+            return startDate.isBetween(moment(), lastDate, 'day') || startDate.isSame(moment(), "day");
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'moment' is not defined.